docs/basics/accounts.md

```diff
@@ -56,80 +56,97 @@ To derive `PubKey`s and `PrivKey`s, the Cosmos SDK uses a standard called [BIP32
                                  +-------------------+
 ```
 
-In the Cosmos SDK, accounts are stored and managed via an object called a [`Keyring`](#keyring).
+In the Cosmos SDK, keys are stored and managed via an object called a [`Keyring`](#keyring).
 
-## Keyring
 
-A `Keyring` is an object that stores and manages accounts. In the Cosmos SDK, a `Keyring` implementation follows the `Keyring` interface:
+## Keys, accounts, addresses and signatures
```

Comment:
```suggestion
## Keys, accounts, addresses, and signatures
```

Reply:
are you sure we should add a comma before `and`?

Reply:
yup

Reply:
so says Grammarly. Our tech writer should leave a review shortly though

Reply:
Thanks @barriebyron . It make sense. Interestingly enough, it seams that this rule applies only when we list at least 3 items: https://www.grammarly.com/blog/comma-before-and/